src/script/components/MessagesList/Message/ContentMessage/FailedToSendWarning/FailedToSendWarning.styles.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react';
+
+export const warning: CSSObject = {color: 'var(--danger-color)'};
```

Comment:
```suggestion
export const warning: CSSObject = {color: 'var(--danger-color)', fontSize: 'var(--font-size-small)'};
```

this is going to be shared by all warnings, we could maybe change the file structure to `.../Message/ContentMessage/Warnings/FailedToSendWarning/FailedtToSendWarning.tsx` with this specific style at `.../ContentMessage/Warnings/Warnings.styles.ts` ?

Reply:
That's good call :)